service/src/routes/api.ts

```diff
@@ -0,0 +1,62 @@
+import express from 'express'
+import dotenv from 'dotenv'
+import axios, { AxiosInstance, AxiosResponse }from 'axios'
+
+export const register = (app: express.Application) => {
+
+    //Get a list of tasks from Mavenlink
+    app.get("/tasks", async (req, res) => {
+        const options = {
+            headers:{
+                Authorization: "Bearer " + process.env.MAVENLINK_TOKEN
+            }
+        };
+        axios.get(process.env.MAVENLINK_TASK_URL, options)
+        .then(response => {
+            console.log(response.data.stories);
+        })
+        .catch(error => console.log(error));
+    });
+
+    //Create a task in Mavenlink
+    app.get("/task/create", async (req, res) => {
```

Comment:
1. Since we are doing `/tasks` for the get tasks, can we keep the URI uniform by doing `/tasks/create` here? Same for put and delete
2. Following the uniformity idea, could we do post, put and delete as HTTP methods for our `/create`, `/update` and `/delete` respectively?

Reply:
Sure, I made the changes.